Homework_25/script.js

```diff
@@ -0,0 +1,44 @@
+function isObjectEmpty(obj) {
+  if (Object.keys(obj).length == 0) {
+    console.log('empty');
+    return true;
+} else {
+  console.log("isn't empty");
+}
+}
+
+let obj = {};
+
+isObjectEmpty(obj);
+
+let user = {
+  name: "Stas",
+  age: 22,
+  sayHello: function () {
+    alert(`Привіт я ${this.name} мені ${this.age} років`);
+  },
+};
+
+user.sayHello();
+
+let Calc = {
+    a:null,
+    b:null,
+  ask: function () 
+  {
+    this.a = +prompt("Напишіть перше число");
+    this.b = +prompt("Напишіть друге число");
+  },
+  sum: function () 
+  {
```

Comment:
in JS convention `{` should right after function declaration:
```
function() {
...
}
```

I know that in C# for example is another convention and there code is written like in your PR, but for JS it's a mistake

Reply:
I solved it all